src/main/java/gateways/Tasks/TaskDatbaseGateway.java

```diff
@@ -0,0 +1,103 @@
+package gateways.Tasks;
+
+import entities.Task;
+import entities.User;
+import gateways.DatabaseGateway;
+import gateways.UserDatabaseGateway;
+import services.CurrentUserService;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskDatbaseGateway extends DatabaseGateway implements TaskDataAccessInterface {
+
+
+    private final CurrentUserService currentUserService;
+    private UserDatabaseGateway userDatabaseGateway;
```

Comment:
Again, need to have an instance of DataAccessInterface<User>.

Reply:
The DataAccessInterface currently doesn't have a save method, so I have no way of persisting the data once changes are made. I'll remember to make these changes once this has been added (I think you also mentioned this during our meeting today).